### PR TITLE
Link 16/actions/edit.svg to document-edit.svg

### PR DESCRIPTION
**Pull request by lakshayrohila**

### CODE REVIEW

File: theme.conf [hunk 1]
```diff
@@ -894,7 +894,7 @@
 24/apps chrome.svg
 24/apps chromium.svg
 24/apps book_dictionary.svg accessories-dictionary.svg
-24/apps edit.svg accessories-text-editor.svg leafpad.svg
+24/apps edit.svg accessories-text-editor.svg leafpad.svg16/actions edit.svg document-edit.svg
```
Comment (01micko) [inline theme.conf]: mistake here

Comment (lakshayrohila) [reply theme.conf]: Was an earlier mistake. Check now, it has been fixed.

Comment (01micko) [reply theme.conf]: Github (or I) was too slow!

Comment (lakshayrohila) [reply theme.conf]: None were slow, I suppose. Github doesn't reload changes in files on its own if files tab is already open, I think.

**Comment by lakshayrohila** [comment]
@01micko, when commit history is not important, you can use **Squash and Merge** feature to merge PRs. It packs the whole PR into 1 single commit and then merge. Doesn't matter though, but still keeps the commit history small :+1:.

You can find more about merging PRs [here](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/about-pull-request-merges).

**Pull request merged**